source/manual/howto-change-a-government-following-an-election.html.md

```diff
@@ -46,39 +53,20 @@ The new government should also instantly appear in the response from
 
 ### Applying a banner to political content published by the previous government
 
-Content in Whitehall can be marked as political. Political content
-which was published under a previous government is
-[historic?][historic]. This is highlighted on the page, and in search
-results. See these pages for example.
+Content in Whitehall can be marked as political. Political content which was
+published under a previous government is [`historic?`][historic]. This is
+highlighted on the page, and in search results. This feature is also known as
+_history mode_.
+
+See these pages for example:
 
 [A content page](https://www.gov.uk/government/speeches/the-issuing-withdrawal-or-refusal-of-passports)
 [A search results page](https://www.gov.uk/search/all?keywords=The+issuing%2C+withdrawal+or+refusal+of+passports&order=relevance)
 
 [historic]: https://github.com/alphagov/whitehall/blob/e518218355d158bfff036a02e312dda714da0aa6/app/models/edition.rb#L647
 
 Marking content as political can happen automatically through the
-[PoliticalContentIdentifier][]. The `political` flag on an `Edition`
-can also be set manually.
+[PoliticalContentIdentifier][]. The `political` flag on an `Edition` can also
+be set manually.
 
 [PoliticalContentIdentifier]: https://github.com/alphagov/whitehall/blob/master/lib/political_content_identifier.rb
```

Comment:
We also need to:


#### Re-index political content in search

After the government has been closed, we need to re-index content in search. There is a [rake task](https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=whitehall&MACHINE_CLASS=whitehall_backend&RAKE_TASK=search:political) to do this:

Reply:
Oh yes... I've added a section about that.